cmake/kokkos_arch.cmake

```diff
@@ -362,7 +362,10 @@ if(KOKKOS_ARCH_ARMV9_GRACE)
       -msve-vector-bits=128
     )
   else()
-    message(WARNING "Compiler does not support ARMv9 Grace architecture")
+    message(
+      SEND_ERROR
+        "Compiler does not support ARMv9 Grace architecture. You may want to try -DKokkos_ARCH_NATIVE=ON instead"
```

Comment:
It is more that "we do not know the option to explicitly support ARMv9 Grace architecture". But I think it is fine the way it is.

Reply:
That's fair.
Actually we might want to discuss that briefly at the developer meeting tomorrow.
@masterleinad would you please add it to the agenda?

Reply:
Sure